### DISCUSSION TITLE
[Weekly Curation Report]: November 13 - 17

### DISCUSSION THREAD

**Discussion by tabbassidaloii** [wikipathways/wikipathways-help#86] [curation-reports]
### Recent changes

Some curation changes by established WP curators.
Disturbed pathways in Duchenne muscular dystrophy (WP5356) - the changes in title declined (back to Duchenne Muscular Dystrophy)

### Under construction

NA

### New pathways

NA

### Deleted Pathways

NA

### Additional notes

Not much to report.

**Comment by khanspers** [comment]
Thanks @tabbassidaloii! For [WP5356](https://classic.wikipathways.org/index.php/Pathway:WP5356), I noticed the title was updated again. I don't mind the new title as I think it is descriptive and accurate, but I'm curious what other reviewers think. Details:

Previous title: Duchenne Muscular Dystrophy
Updated title: Disturbed pathways in Duchenne muscular dystrophy

Please comment here with any comments on the new pathway title, pro or con.

**Comment by AlexanderPico** [comment]
"Disturbed" more often refers to an emotional or mental state. I'd prefer "Disrupted" or "Affected".

**Reply by Chris-Evelo** [reply]
Then I would go with affected. Not all effects on pathways during disease development need to be negative, which I think is more indicated by disturbed.

**Comment by AlexanderPico** [comment]
Following the series of edits, it seems the primary disagreement is whether DMD is capitalized or not:

> Duchenne muscular dystrophy vs. Duchenne Muscular Dystrophy

Disease names are not capitalized, _but Duchenne is a person's name_. So, I'd vote for:

"Affected pathways in Duchenne muscular dystrophy"

**Reply by tabbassidaloii** [reply]
Agree and thanks for the elaboration.

**Reply by khanspers** [reply]
I updated the title.
